hw06/task1.js

```diff
@@ -0,0 +1,35 @@
+//Function declaration
+
+function calculateRectangle (width1, height1)//function declaration
+{
+    //console.log(sumOfWidthHeight)
+    const calculateRectangleArea = width1 * height1
+    return calculateRectangleArea
+}
+const widthHeight = calculateRectangle (100,450)
+
+console.log(widthHeight)
+
+
+
+//Function Expression
+
+const calculateRectangle2 = function (width2, height2)
+{
```

Comment:
please write open brace in the same row as parameters. The way you do it is not common approach for JS